ci.yml

```diff
@@ -0,0 +1,12 @@
+script:
+        - cd build
+        - echo $PATH
+        - export HOME=$(dirname "$PWD")
+        - oc login -u kubeadmin -p BdTY8-Vra64-gD3cI-dAPS6 --insecure-skip-tls-verify=true https://api.crc.testing:6443
+        - IMAGE_BUILD_ARGS="--storage-driver vfs" ./build_ci.sh
```

Comment:
It's not merged yet, but I'm hoping to eliminate the need for these args. See https://github.com/redhat-service-assurance/service-assurance-operator/pull/43